firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/common/APIController.kt

```diff
@@ -120,6 +122,19 @@ internal constructor(
       throw FirebaseCommonAIException.from(e)
     }
 
+  suspend fun generateImage(request: GenerateImageRequest): GenerateImageResponse =
```

Comment:
Is there no streaming support for imagen yet?

Reply:
correct